lib/registries/addon/drafts/draft/template.hbs

```diff
@@ -2,3 +2,52 @@
 <section>
     {{!-- Fill out this registration form. --}}
 </section>
+<Registries::DraftRegistrationManager
+    @currentPage={{this.model.pageIndex}}
+    @modelTaskInstance={{this.model.taskInstance}}
+    @updateRoute={{route-action 'updateRoute'}}
+    as |draftRegistrationManager|
+>
+    <OsfLayout 
+        @backgroundClass='{{local-class 'ContentBackground'}}'
+        as |layout|>
+        <layout.leftNav as |leftNav|>
+            {{#each draftRegistrationManager.pageManagers as |pageManager index|}}
+                {{#let (hash
+                    active=(eq draftRegistrationManager.currentPage index)
+                    visited=pageManager.isVisited
+                    valid=pageManager.isValid
+                ) as |ctx|}}
+                    <leftNav.link
+                        data-analytics-name='Overview'
+                        @models={{array draftRegistrationManager.draftId}}
+                        @href='/drafts/{{draftRegistrationManager.draftId}}/{{index}}/'
+                        @icon={{if ctx.active 'circle-o' (if ctx.visited (if ctx.valid 'check-circle-o' 'exclamation-circle') 'circle') }}
+                        @label={{pageManager.pageHeadingText}}
+                        local-class={{if ctx.active 'Active' (if ctx.visited (if ctx.valid 'Valid' 'Invalid') 'Unvisited') }}
+                        class={{if ctx.active 'active'}}
+                        as |link|
+                    >
+                        <link.icon
+                            icon={{@icon}}
+                        />
+                        {{#unless this.isCollapsed}}
+                            <link.label
+                                label={{@label}}
+                                local-class={{unless ctx.active (if ctx.visited '' 'UnvisitedDark')}}
```

Comment:
Same here. Also, a single `{{unless}}` block is fine (although I don't like it), but another conditional block nested inside `{{unless}}` makes it even harder to reason.